pennylane/qchem/vibrational/vibrational_class.py

```diff
@@ -150,6 +172,17 @@ def optimize_geometry(molecule, method="rhf"):
     Returns:
         array[array[float]]: optimized atomic positions in Cartesian coordinates
 
+    **Example**
+
+    >>> symbols  = ['H', 'F']
+    >>> geometry = np.array([[0.0, 0.0, 0.0],
+                            [0.0, 0.0,  1.0]])
+    >>> mol = qml.qchem.Molecule(symbols, geometry)
+    >>> eq_geom = qml.qchem.optimize_geometry(mol)
+    >>> eq_geom
+    [[0.0, 0.0, -0.40277116],
+     [0.0, 0.0,  1.40277116]]
```

Comment:
This is what I get:
```
array([[ 0.        ,  0.        , -0.40277116],
       [ 0.        ,  0.        ,  1.40277116]])
```

Reply:
Do we want it with all the spaces in between?

Reply:
Yes.